public/css/styles.css

```
@@ -17,31 +17,32 @@ html, body {
 }
 
 .background {
-  height: 100%;
   width: 100%;
+  height: 600Px;
```

```suggestion
  height: 600px;
```

そもそもこの変更がなぜ必要なのかわからないですが、せめて単位は小文字で書いてほしいです。
styles.cssはいろんなファイルから読み込む全般的なスタイルを書いていきたいので、あまり書き換えてほしくない感じがあります。

---

pxの部分は純粋なミスですね。
cssにbackgroundを足したのは、変に縦幅が伸びないようにです。